java/mykola-melnyk/src/com/zoolatech/lecture4/tasks/_6/StringAnalyze.java

```diff
@@ -0,0 +1,41 @@
+package com.zoolatech.lecture4.tasks._6;
+
+import java.util.Arrays;
+
+/**
+ * Create a method that accepts a string, counts number of letters (assume only English lower and upper letters), digits, spaces and other characters and outputs this data:
+ * Input: “aB?C de 42* dcd”
+ * Output:
+ * 	“8 letters”
+ * 	“2 digits”
+ * 	“3 spaces”
+ * 	“2 other characters”
+ */
+
+public class StringAnalyze {
+    public static void analyze(String input) {
+        int length = input.length();
+        char[] chars = new char[length];
```

Comment:
there's a simple method how to get chars: input.toCharArray()

Reply:
Done